tests/test_build.py

```diff
@@ -510,6 +510,20 @@ def test_build(self):
                 .qualify("row_number() OVER (PARTITION BY a ORDER BY b) = 1"),
                 "SELECT * FROM table QUALIFY ROW_NUMBER() OVER (PARTITION BY a ORDER BY b) = 1",
             ),
+            (lambda: exp.delete("tbl", "x = 1").from_("table"), "DELETE FROM table WHERE x = 1"),
+            (
+                lambda: exp.delete("tbl", "x = 1").from_("table").where("y = 2"),
+                "DELETE FROM table WHERE x = 1 AND y = 2",
+            ),
+            (lambda: exp.Delete(this="tbl"), "DELETE FROM tbl"),
```

Comment:
this is not correct usage, this is a string and needs to be a table. this is why you should only use the helper and not the class